Fix: bugfixes and overall improvements to alert history

ahmadshaheer opened the pull request:
### Summary
- [x] 1. remove the severity label from the top level (this is redundant because you are already showing the severity with special colours)


![Screenshot 2024-09-02 at 10 17 44 PM](https://github.com/user-attachments/assets/4d609aaa-9619-4a41-b153-644c8fad84bb)


- [x] 2. I find the alert number of redundant


![Screenshot 2024-09-02 at 10 18 57 PM](https://github.com/user-attachments/assets/1bff0dce-16bc-4495-a4a5-7af50d23962a)

- [x] 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert


![Screenshot 2024-09-02 at 10 20 34 PM](https://github.com/user-attachments/assets/13b80ea4-1db0-4af0-8ae1-7d02f0871f02)

- [x] 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.


after disable

![Screenshot 2024-09-02 at 10 20 34 PM](https://github.com/user-attachments/assets/8d4e3e2e-7a93-4675-9b8f-5fca94c19244)

after refresh

![Screenshot 2024-09-02 at 10 24 44 PM](https://github.com/user-attachments/assets/3bfc37bc-deff-48d9-9aa5-e07a200231a5)

- [x] 6. make the default period 6h in history


- [x] 7. change the top contributors' description to
"Top contributors highlight the most frequently triggering group-by attributes in multi-dimensional alerts" because not every alert will need to have group-by.

- [ ] 8. Search doesn't work - we should either remove it or make it functional

- [x] 9. The timeline doesn't show all the items (I am not sure what the logic but it should show all the items in /overall_status)

- [x] 10. Let's not show the value column for now

- [x] 11. on the next page offset should be the total consumed so far but `1` is sent. It should be like 20, 40, etc..

- [x] 12. The logs link shows "View traces" and the traces link shows "View logs" link
<!-- ✍️ A clear and concise description...-->

#### Related Issues / PR's

<!-- ✍️ Add the issues being resolved here and related PR's where applicable  -->

#### Screenshots

NA

<!-- ✍️ Add screenshots of before and after changes where applicable-->

#### Affected Areas and Manually Tested Areas

<!-- ✍️ Add details of blast radius and dev testing areas where applicable-->

YounixM commented on frontend/src/container/AlertHistory/Timeline/Table/Table.styles.scss:
```diff
@@ -88,6 +91,9 @@
 		}
 	}
 	.alert-history-label-search {
+		// .ant-select-selection-placeholder {
```
Remove this

ahmadshaheer replied:
Done

YounixM commented on frontend/src/container/AlertHistory/Timeline/Table/Table.tsx:
```diff
@@ -1,22 +1,22 @@
-import { useMemo, useState } from 'react';
+import {
+	useMemo,
+	// , useState
```
Remove commented code

ahmadshaheer replied:
Done

sharpshooter90 commented:
@ahmadshaheer please got through the source design once, and reflect on the comments you have shared: https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2733-15237&t=ra6jnNGgu5FFNsFn-4

sharpshooter90 commented:
> 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert
A: add the description on hover, switch is clear enough to communicate what is it. History is inside the tab in lower level. There wont be any confusion. 

 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.
 A: @ahmadshaheer this is a functionality problem not a design issue.

SagarRajput-7 commented on frontend/src/container/AlertHistory/Timeline/Table/Table.tsx:
```diff
@@ -1,31 +1,25 @@
 		ruleId,
-	} = useGetAlertRuleDetailsTimelineTable({ filters });
+	} = useGetAlertRuleDetailsTimelineTable();
+	// { filters }
```
please make sure to remove all the unwanted comments once when the final PR against develop is ready

ahmadshaheer replied:
Done

ahmadshaheer commented:
> > 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert
> >    A: add the description on hover, switch is clear enough to communicate what is it. History is inside the tab in lower level. There wont be any confusion.
> 
> 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.
>    A: @ahmadshaheer this is a functionality problem not a design issue.

@sharpshooter90 , 
> 3.
I have added a tooltip

> 4.

Yeah, I fixed this issue
